modular_zubbers/code/modules/pai/ghost_pai.dm

```diff
@@ -0,0 +1,27 @@
+/obj/item/pai_card/attack_ghost(mob/user)
+	ghost_activate(user)
+
+/**
+ * Checks if the user is allowed to be a pAI, and if so, creates a new pAI mob.
+ * Assuming the pAI card itself is empty of a personality.
+ */
+/obj/item/pai_card/proc/ghost_activate(mob/user)
+	if(pai)
+		return FALSE
+	var/pai_ask = tgui_alert(user, "Become a pAI? (Warning, You can no longer be revived, and all past lives will be forgotten!)", "Confirm", list("Yes","No"))
+	if(pai_ask != "Yes" || QDELETED(src))
+		return
+	var/pai_ckey = user.ckey
+	if(!user.client || !isobserver(user) || is_banned_from(pai_ckey, ROLE_PAI))
+		return FALSE
+	if(!SSpai.candidates[pai_ckey])
+		to_chat(user, span_warning("Personality Matrix Corrupted! Please reload Personality file (Open the pAI submit window and click the load button before trying again)"))
```

Comment:
errr, is this the error message for when someone clicked yes but doesn't have PAI settings set up?

Reply:
> errr, is this the error message for when someone clicked yes but doesn't have PAI settings set up?

Aye